sources/zh.mangabz/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "mangabz"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+aidoku = { git = "https://github.com/Aidoku/aidoku-rs.git", version = "0.3.0", features = [
+    "json",
+] }
+regex = { version = "1.11.1", default-features = false, features = ["unicode"] }
+
+[dev-dependencies]
+aidoku = { git = "https://github.com/Aidoku/aidoku-rs.git", features = [
+    "test",
+] }
+aidoku-test = { git = "https://github.com/Aidoku/aidoku-rs.git" }
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.dev]
+panic = "abort"
+
+[profile.release]
+panic = "abort"
+opt-level = "s"
+strip = true
+lto = true
```

make sure there's an extra newline here for consistency with the other sources
```suggestion
lto = true

```